Mobile Version Finished

mm-pazouki opened the pull request:
Added portfolio HTML content.
Added CSS style for a mobile version.
Added Images and Icons of the project.
Added JS files.
Added nav bar.

mm-pazouki committed:
Update README.md

Mobile Version Finished yesterday and today I just want to accept it for the progress.

Johnadibe reviewed (commented):
Hi @mm-pazouki, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not have a short summary describing the introduced changes in more detail to this PR. Please provide a descriptive summary describing the introduced changes. Please ensure to also use the correct GitHub flow.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

tsohleDev commented on README.md:
- [x] I Suggest that you follow the GitHub flow on every PR you open, thus don't merge before you get approved, because merging before could introduce bugs into a bug free working code base 😉

tsohleDev commented on README.md:
- [x] I suggest that you stick to the design and use the **typography** used in the Figma design e.g `Playfair Display` is used in the `h1` 👍 
- [x] And on that note I think it would have been if you have used `h2` or `h3` not `h1` so that you use semantic tags, and your HTML becomes industry standard
- [x] Your `p` tag that describes you in the home section has a big font size, an thus does not match the Figma, I think it would be better to reduce it so that it can be as close as possible to the design
- [x] Note that the decoration is overlaying with the project, I kindly suggest you fix this 😉 
- [x] I think it would be better if you didn't make the project titles `bold` is they are not in the design

### It is very important for a front end developer to be able to re-create the design as close as possible, **pixel perfect** would be even better, so i suggest you make these changes in attempt of that

| Site | Figma |
| --- | --- |
| ![yours](https://user-images.githubusercontent.com/102636489/234996093-167523e1-d071-47ec-b920-ce031a68fb63.png) | ![figma](https://user-images.githubusercontent.com/102636489/234996144-7ee637d9-c85a-4afd-97b6-67299a9f72c0.png) |
| ![yours](https://user-images.githubusercontent.com/102636489/234999741-89ae169c-b0da-4b10-8c45-796a98485d10.png) | ![figma](https://user-images.githubusercontent.com/102636489/234999787-8cf7f75a-ca96-430c-8226-e79c367178fa.png) |

tsohleDev commented on README.md:
## README

- [x] I think it would be better if you would include **key features** and *future features**, as this makes it easy for other developers and recruiters to understand your project, and maybe get some stars or a job 😄 
- [x] I kindly suggest you also add the **Getting Started** section, as to let other developers or recruiters who might not know how to use GitHub or vs code how to use your application, because we want to be accommodative to all 😉

tsohleDev reviewed (changes requested):
## Hi @mm-pazouki 

Good job so far!
There are some issues that you still need to work on.

![163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856](https://user-images.githubusercontent.com/97100091/191558701-8ed50201-3123-435c-89d8-29edcb66c64d.gif)

## Highlights!
- Linters ✔️ 
- Descriptive pull request ✔️ 
- README template ❌ 
- GitHub Flow ❌
- CSS ❌
- Semantic Tags ❌ 

Hello ✋, you have done well creating your project 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Compliment
_Every comment with the [COMPLIMENT] prefix is not  a request, just a compliment at your skills_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@tsohleDev` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mm-pazouki committed:
Headers changed
P font size fixed.
decoration fixed
bold title removed
key features and starter method not include because it is personal portfolio and I dont think this project would be useful for anyone. and other thing what you want from portfolio? key features? yes it can buy bread every morning :|

mm-pazouki committed:
Linter fixed

shella12 reviewed (commented):
Hi @mm-pazouki , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some missing sections from README template. Please use this [template](https://github.com/microverseinc/readme-template) and make sure you have added all the missing main sections (i.e. Acknowledgment, contribution, support, future feature). The only section that is OPTIONAL is FAQ.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

mm-pazouki commented:
https://github.com/shella12/portofolio-ayesha/blob/main/README.md
 so why is yours not similar to the template you mentioned?

shella12 commented:
It's because the template then was different from this one

Ayesha Arshad

On Fri, Apr 28, 2023, 10:10 PM Mohammad Mehdi Pazouki <
***@***.***> wrote:

> https://github.com/shella12/portofolio-ayesha/blob/main/README.md
> so why is yours not similar to the template you mentioned?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mm-pazouki/Portfolio/pull/2#issuecomment-1527849593>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKVZAXC7YWGRGSSGYG7NH63XDP22DANCNFSM6AAAAAAXODAPUA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

stanleySimeon commented on README.md:
Kindly I suggest you remove the live demo section, if you don't have deployed the project yet! Following this step can make your project look more professional. 😄

stanleySimeon commented on Index.html:
```diff
@@ -1,179 +1,283 @@
+        </section>
+
+        <section id="works" class="works">
+            <h2 class="h2 portfolio__hearder">Projects</h2>
+            <article class="portfolio project__one">
+                <img src="./Contents/Resources/Pictures/Correspondence_img.jpg" alt="project one is a web site" class="project__photo project__onePhoto">
+                <div class="project__info hover__one">
+                    <h3 class="project__title">Correspondence</h3>
+                    <ul class="project__stack__container">
+                        <li class="project__stack stack1 one__stacks">HTML/CSS</li>
+                        <li class="project__stack stack3 one__stacks">JavaScript</li>
+                        <li class="project__stack stack3 one__stacks">SQL</li>
+                    </ul>
+                    <div class="project__button__holder">
+                        <button class="buttons project__button">
+                            See this project
+                            <span class="icon arrow material-symbols-outlined">
+                                arrow_right_alt
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </article>
```
According to the Figma template, you should Follow the Design to make your project exactly like the Design, Kindly I suggest you follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=X7Yh3FuNhyLhpYDE-0) to make your project match the design template.

> ### The Figma Design
>
><img width="182" alt="Screenshot 2023-04-28 at 15 24 05" src="https://user-images.githubusercontent.com/91237525/235236095-8184ee77-2593-4e18-8d3b-324d08907f48.png">
>
>
>### Your Design
>
><img width="282" alt="Screenshot 2023-04-28 at 15 24 24" src="https://user-images.githubusercontent.com/91237525/235236202-a80cf124-526e-47d7-8f9a-f49939d928eb.png">

Kindly, I suggest you use the font-size, letter-spacing, as it is in the Figma template.

stanleySimeon commented on Index.html:
```diff
@@ -1,179 +1,283 @@
+                        <li class="about__li">Python</li>
+                        <li class="about__li">HTML</li>
+                        <li class="about__li">CSS</li>
+                    </ul>
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/Sharp.svg" alt="" class="pound__logo">
+                    </div>
+                </li>
+
+                <li class="frameworks about__lists">
+                    <h3 class="h3">Frameworks</h3>
+                    <ul class="about__ul">
+                        <li class="about__li">Bootstrap</li>
+                        <li class="about__li">Django</li>
+                        <li class="about__li">Rest</li>
+                        <li class="about__li">Celery</li>
+                    </ul>
+
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/Rectangle 19.svg" alt="" class="square__logo">
+                    </div>
+                </li>
+
+                <li class="skills about__lists">
+                    <h3 class="h3">Skills</h3>
+                    <ul class="about__ul">
+                        <li class="about__li">Postgres</li>
+                        <li class="about__li">RabbitMQ</li>
+                        <li class="about__li">GitHub</li>
+                        <li class="about__li">Linux</li>
+                        <li class="about__li">Docker</li>
+                    </ul>
+
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/ic_skills.svg" alt="" class="idea__logo">
+                    </div>
+                </li>
+
+                <img src="./Contents/Resources/Icons/image_dots_with_teal.svg" alt="" class="about__deco">
+            </ul>
```
According to the Figma template, you should Follow the Design to make your project exactly like the Design, Kindly I suggest you follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=X7Yh3FuNhyLhpYDE-0) to make your project match the design template.

> ### The Figma Design
>
><img width="194" alt="Screenshot 2023-04-28 at 15 31 27" src="https://user-images.githubusercontent.com/91237525/235237297-30ca7512-49f3-48a1-81bc-2d3373fcfe27.png">
>
>
>### Your Design
>
><img width="188" alt="Screenshot 2023-04-28 at 15 31 44" src="https://user-images.githubusercontent.com/91237525/235237344-665fc81d-c0f2-441b-b798-119205231fe8.png">


Kindly, I suggest you use the font size, letter-spacing, margin, and padding as it is in the Figma template.

stanleySimeon commented on Index.html:
```diff
@@ -1,179 +1,283 @@
-      </li>
-    </ul>
-  </section>
-  <form action="https://formspree.io/f/xeqdoajj" method="POST" id="contact-form">
-    <h1>Get Started</h1>
-    <p>I Prefer To Speak By My Gestures.</p>
-    <fieldset>
-      <label for="customer-name">
-        Your name <span class="errorText"></span>
-        <br />
-        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
-      </label>
-    </fieldset>
-    <fieldset>
-      <label for="customer-email">
-        Your email <span class="errorText"></span>
-        <br />
-        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
-      </label>
-    </fieldset>
-    <span class="errorText"></span>
-    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
-      required></textarea>
-    <span class="errorText" id="submitError"></span>
-    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
-  </form>
```
Kindly, I suggest you follow the template as it is, to make your project look professional. It's highly recommended to show the label for each input and text-area.
Kindly make sure you fix these changes to make your project match the template.

>### The Figma Design
>
><img width="166" alt="Screenshot 2023-04-28 at 15 37 15" src="https://user-images.githubusercontent.com/91237525/235238390-1f246616-be67-4815-94d7-620832b66109.png">
>
>
>### Your Design
>
><img width="186" alt="Screenshot 2023-04-28 at 15 37 06" src="https://user-images.githubusercontent.com/91237525/235238467-03fc6e28-69c0-4a7b-9035-8de8fa3b36b5.png">

stanleySimeon reviewed (changes requested):
# Hi @mm-pazouki 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media3.giphy.com/media/5n1XqQh0DskgKMKejY/giphy-downsized.gif?cid=29caca75bxvcwozwzif8zdc9c1flo4llwfq3vy73onbz5z7p&ep=v1_gifs_search&rid=giphy-downsized.gif&ct=g
)

## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

* [x] As mentioned by the previous reviewer, I think it would be better if you would include key features and future features*, as this makes it easy for other developers and recruiters to understand your project, and maybe get some stars or a job 😄

* [x] As mentioned by the previous reviewer, kindly I suggest you also add the Getting Started section, as to let other developers or recruiters who might not know how to use GitHub or vs code how to use your application, because we want to be accommodative to all 😉

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mm-pazouki committed:
Readme fixed.
Form placeholder fixed.
About fixed.

shella12 reviewed (commented):
Hi @mm-pazouki, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [x] has some missing sections from README template. Please use this [template](https://github.com/microverseinc/readme-template) and make sure you have added all the missing main sections (i.e. support). 
- [x] Doesn't have a descriptive "Getting Started" section. Please add a bit about how to run, install or use your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

mm-pazouki committed:
readme fixed

jmonto55 commented on README.md:
````diff
@@ -66,18 +103,71 @@
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails server
+```
````
- [x] [OPTIONAL] Great work implementing the changes requested inside your README file 💯. However, please kindly ensure that you give the correct instructions to run your project. Since this is not a rails application, I would kindly suggest you use more appropriate instructions here like Open using Live Server 🤓.

jmonto55 commented on README.md:
````diff
@@ -66,18 +103,71 @@
 
+Example command:
+
+```sh
+  cd Portfolio
+  gem install
+```
````
- [x] [OPTIONAL] Good work implementing the changes requested by the previous reviewers ✔️. However, please kindly remove the gem install command since this is not a Ruby or a Rails application 🤓.

jmonto55 commented on README.md:
```diff
@@ -109,19 +199,40 @@ Feel free to check the [issues page](../../issues/).
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```
- [x] [OPTIONAL] Excellent job including your License inside your README file 🚀. However, when I click the link it takes me to a 404 not found page, please kindly ensure to fix this by changing line 236 back to `
This project is [MIT](./MIT.md) licensed.` 🤓.

jmonto55 reviewed (approved):
# STATUS: APPROVED 🟢

Hi @mm-pazouki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![AgreeSealGIF](https://user-images.githubusercontent.com/116701485/232347692-8a213af9-b250-4d2c-a690-7f42d3c630e0.gif)


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
